marklogic-data-hub-central/ui/src/components/common/hc-tag/hc-tag.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import {render, screen, cleanup} from "@testing-library/react";
+import styles from "../../../ruleset-multiple-modal.module.scss";
+import {fireEvent} from "@testing-library/react";
+import HCTag from "./hc-tag";
+
+afterEach(() => {
+  cleanup();
+});
+
+test("should render a Tag component ", () => {
+  render(<HCTag label={"test tag"}>This is a tag check</HCTag>);
+  const tagElement = screen.getByTestId("tag-component");
+  expect(tagElement).toBeInTheDocument();
+});
+test("should render a Tag component ", () => {
```

Comment:
@MSBrignoni is it possible to change this `test` clause title? Something along the lines of `"should render a customized Tag component"` so it is not the same as the previous one above. Thanks!

Reply:
Thank you Mathias!, ready.